backend/Schema/auth/signup.js

```diff
@@ -0,0 +1,35 @@
+const schema = {
+  required: ['username', 'email_id', 'name', 'password', 'department'],
+  properties: {
+    username: {type: 'string'},
+    email_id: {type: 'string', format: 'email'},
+    name: {type: 'string'},
+    password: {type: 'string', format: 'password'},
+    branch: {type: 'number'},
+    department: {type: 'number'},
+    admission_no: {type: 'string', minLength: 8, maxLength: 8},
```

Comment:
I would suggest to use regex here. 2 numbers followed by 2 letters followed by 4 numbers is the general admission number format.